repository/s.json

```diff
@@ -3722,6 +3722,20 @@
 				}
 			]
 		},
+		{
+			"name": "Status Bar JsonPath",
+			"details": "https://github.com/akirk/StatusBarJsonPath",
+			"author": "Alex Kirk",
+			"labels": ["status bar", "json", "jsonpath"],
+			"readme": "https://gitlab.com/akirk/StatusBarJsonPath/-/raw/master/README.md",
+			"issues": "https://gitlab.com/akirk/StatusBarJsonPath/-/issues",
```

Comment:
your issues and readme are in the default location so you don't have to list them here